init.lua

```diff
@@ -77,16 +77,24 @@ local function parti2(pos)
 end
 
 -- Teleport Request System
+local function clear_tpr_list(name)

```

Comment:
Suggestion:
```
function clear_tpr_list(name)
```
This will make it compatible with other mods which use this function.